nodes/mainnet/src/cli.rs

```diff
@@ -61,9 +61,10 @@ pub enum Subcommand {
 	#[clap(subcommand)]
 	Benchmark(Box<frame_benchmarking_cli::BenchmarkCmd>),
 
-	/// Try some command against runtime state.
-	#[cfg(feature = "try-runtime")]
-	TryRuntime(try_runtime_cli::TryRuntimeCmd),
+	/// Try-runtime has migrated to a standalone
+	/// [CLI](<https://github.com/paritytech/try-runtime-cli>). The subcommand exists as a stub and
+	/// deprecation notice. It will be removed entirely some time after Janurary 2024.
+	TryRuntime,
```

Comment:
On the line 71 we already have this property. Please remove one. It fails to compile because of this.